src/vs/editor/contrib/snippet/common/snippet.ts

```diff
@@ -179,296 +211,296 @@ interface IParsedLine {
-			}
-
-			//This is an escape sequence or not a special character, just insert it
-			placeHolderStack[placeHolderStack.length - 1].placeHolderText += line.charAt(i);
-			++resultIndex;
-			++i;
-		}
-
-		// Sort the placeholder in order of apperance:
-		placeHolders.sort((a, b) => {
-			if (a.startColumn < b.startColumn) {
-				return -1;
-			}
-			if (a.startColumn > b.startColumn) {
-				return 1;
-			}
-			if (a.endColumn < b.endColumn) {
-				return -1;
-			}
-			if (a.endColumn > b.endColumn) {
-				return 1;
-			}
-			return 0;
-		});
-
-		return {
-			line: placeHolderStack[0].placeHolderText,
-			placeHolders: placeHolders
-		};
-	}
-};
-
-const EmmetSnippetParser = new class implements ISnippetParser {
-
-	parse(template: string): CodeSnippet {
-		template = _convertExternalSnippet(template, ExternalSnippetType.EmmetSnippet);
-		return InternalFormatSnippetParser.parse(template);
-	}
-};
-
-export enum ExternalSnippetType {
-	TextMateSnippet,
-	EmmetSnippet
-}
-
-// This is used for both TextMate and Emmet
-function _convertExternalSnippet(snippet: string, snippetType: ExternalSnippetType): string {
-	var openBraces = 0;
-	var convertedSnippet = '';
-	var i = 0;
-	var len = snippet.length;
-
-	while (i < len) {
-		var restOfLine = snippet.substr(i);
-
-		// Cursor tab stop
-		if (/^\$0/.test(restOfLine)) {
-			i += 2;
-			convertedSnippet += snippetType === ExternalSnippetType.EmmetSnippet ? '{{_}}' : '{{}}';
-			continue;
-		}
-		if (/^\$\{0\}/.test(restOfLine)) {
-			i += 4;
-			convertedSnippet += snippetType === ExternalSnippetType.EmmetSnippet ? '{{_}}' : '{{}}';
-			continue;
-		}
-
-		// Tab stops
-		var matches = restOfLine.match(/^\$(\d+)/);
-		if (Array.isArray(matches) && matches.length === 2) {
-			i += 1 + matches[1].length;
-			convertedSnippet += '{{' + matches[1] + ':}}';
-			continue;
-		}
-		matches = restOfLine.match(/^\$\{(\d+)\}/);
-		if (Array.isArray(matches) && matches.length === 2) {
-			i += 3 + matches[1].length;
-			convertedSnippet += '{{' + matches[1] + ':}}';
-			continue;
-		}
-
-		// Open brace patterns placeholder
-		if (/^\${/.test(restOfLine)) {
-			i += 2;
-			++openBraces;
-			convertedSnippet += '{{';
-			continue;
-		}
-
-		// Close brace patterns placeholder
-		if (openBraces > 0 && /^}/.test(restOfLine)) {
-			i += 1;
-			--openBraces;
-			convertedSnippet += '}}';
-			continue;
-		}
-
-		// Escapes
-		if (/^\\./.test(restOfLine)) {
-			i += 2;
-			if (/^\\\$/.test(restOfLine)) {
-				convertedSnippet += '$';
-			} else {
-				convertedSnippet += restOfLine.substr(0, 2);
-			}
-			continue;
-		}
-
-		// Escape braces that don't belong to a placeholder
-		matches = restOfLine.match(/^({|})/);
-		if (Array.isArray(matches) && matches.length === 2) {
-			i += 1;
-			convertedSnippet += '\\' + matches[1];
-			continue;
-		}
-
-		i += 1;
-		convertedSnippet += restOfLine.charAt(0);
-	}
-
-	return convertedSnippet;
-};
+// const InternalFormatSnippetParser = new class implements ISnippetParser {
```

Comment:
well, you mentioned we shouldn't "delete" the code yet, so I commented it instead :)